storage/migrations/0011_emerald_incoming.up.sql

```diff
@@ -0,0 +1,57 @@
+BEGIN;
+
+CREATE TABLE block_extra (
+                             chain_alias VARCHAR(32),
```

Comment:
what is this indentation

Reply:
Weirdness indeed.

Assuming you used an autoformatter and you're not happy with it: I looked around and tested a few; sqlfmt by Cockroach Labs looks best. See [writeup](https://www.cockroachlabs.com/blog/sql-fmt-online-sql-formatter/), [demo](https://sqlfum.pt/) (increase max width with non-obvious GUI slider), [binary](https://github.com/mjibson/sqlfmt/releases/tag/v0.4.0).

I don't feel motivated enough to integrate it into our CI, but it's where we should head eventually, probably.

Reply:
shouldn't have to worry about this in this PR, gonna address in #147